app/models/reservation.rb

```diff
@@ -0,0 +1,22 @@
+class Reservation < ActiveRecord::Base
+  belongs_to :request
+  has_one :donor
+
+  state_machine :initial => :unreserved do
```

Comment:
The arrows actually seem to be a convention enforced by the state machine gem itself. I tried changing it to colons, but it resulted in errors :cry: